test/data/marriage-abroad-responses-and-expected-results.yml

```diff
@@ -22931,7 +22931,7 @@
   - uk
   - partner_british
   - opposite_sex
-  :next_node: :outcome_opposite_sex_marriage_in_other_countries
+  :next_node: :outcome_opposite_sex_marriage_in_yemen
```

Comment:
I'm slightly confused as to why `yemen` does not appear in the responses for any of these changes to the `marriage-abroad-responses-and-expected-results.yml` file.